pkgs/top-level/all-packages.nix

```diff
@@ -18580,8 +18580,22 @@ in
       bzip2 = null;
     };
   };
+  systemdMinimal = systemd.override {
+      withResolved = false;
+      withLogind = false;
+      withHostnamed = false;
+      withLocaled = false;
+      withTimedated = false;
+      withHwdb = false;
+      withEfi = false;
+      withImportd = false;
+      withCryptsetup = false;
+      cryptsetup = null;
+      lvm2 = null;
+    };
+
 
-  udev = systemd; # TODO: move to aliases.nix
+  udev = systemdMinimal; # TODO: move to aliases.nix
```

Comment:
```suggestion
  udev = systemdMinimal;
```

Reply:
Also, I'm not sure if we should change this inside this PR already - it might cause some breakages. Maybe just drop that change for now.

Reply:
That todo is not from me. If we don't change it now, will somebody remember to change it? Should I change the TODO to change to systemdMinimal?

Reply:
yeah, sounds good.